watertap/unit_models/electrodialysis_1D.py

```diff
@@ -713,20 +721,22 @@ def _make_performance(self):
             self._pressure_drop_calculation()
 
             @self.Constraint(
+                self.flow_channel_set,
                 self.flowsheet().time,
-                self.diluate.length_domain,
-                doc="Express deltaP_term by the calculated pressure drop data, diluate.",
-            )
-            def eq_deltaP_diluate(self, t, x):
-                return self.diluate.deltaP[t, x] == -self.pressure_drop[t]
-
-            @self.Constraint(
-                self.flowsheet().time,
-                self.diluate.length_domain,
-                doc="Express deltaP_term by the calculated pressure drop data, concentrate.",
+                self.difference_elements,
+                doc="Express deltaP_term by the calculated pressure drop data.",
             )
-            def eq_deltaP_concentrate(self, t, x):
-                return self.concentrate.deltaP[t, x] == -self.pressure_drop[t]
+            def eq_deltaP(self, c, t, x):
+                if x not in self.difference_elements:
+                    self.concentrate.deltaP[t, x].fix(0)
+                    self.diluate.deltaP[t, x].fix(0)
+                else:
+                    if c == "concentrate":
+                        return (
+                            self.concentrate.deltaP[t, x] == -self.pressure_dx[c, t, x]
+                        )
+                    elif c == "diluate":
+                        return self.diluate.deltaP[t, x] == -self.pressure_dx[c, t, x]
```

Comment:
For 1D, deltaP is essentially dP_dx, as is pressure_dx. It seems redundant to do this, but maybe I am missing something. Should `pressure_dx` instead just be a reference of `deltaP`?